src/_help/continuous-integration/github-actions.md

```diff
@@ -7,35 +7,48 @@ title: GitHub Action
 
 ## Generate your API documentation & changelog
 
-Bump is a Continuous Documentation Platform: it lets you keep your API doc always synchronized with your codebase. With this [Github Action](https://github.com/actions) you can automatically generate your API reference (with changelog and diff) on [Bump](https://bump.sh) from any [OpenAPI](https://github.com/OAI/OpenAPI-Specification) or [AsyncAPI](https://github.com/asyncapi/asyncapi) file.
+Bump.sh helps you build a branded single source of truth, cataloging all your APIs. We’ve created the reference point for teams consuming and building APIs, no matter which technology they rely on.
```

Comment:
Not sure about that part. I don't think that we should promote Bump.sh at this step.